### PR TITLE
Proof GROUND_TRACKING_DETERMINATION_PROGRAM #586

### PR DESCRIPTION
**Pull request by rcornbtree**
Full file checked, another eye might be nice on:

- Generally wonky indents + line breaks across first page
- Line 76: Underscore used where there should be a negative
- Line 115: Possible that '1ST' is typed 'IST', unsure
- Line 161: Unsure what character is used, left unchanged from '+'

### CODE REVIEW

**Review by wopian — CHANGES_REQUESTED** [review]
Thank you for opening your first proofing PR for Apollo 11 🎉 

I've gone through and double checked the PR changes, marked any missing transcription errors as commit suggestions (some were out of range of touched lines so was only able to create comments showing the changes needed) and hopefully answered your questions.

> Generally wonky indents + line breaks across first page

If this is rereferring to the AXO/AYO table, I've fixed the alignment issues for you. 

Commented tables will always be a bit wider or narrower as we're working with an additional `# ` at the front of every comment line that the source files do not need to indicate comments.

This sometimes forces an additional tab to keep the vertical alignment as it effectively turns the first 8-character wide tab into 6 usable characters, so what would fit inside one tab column now takes up two tab columns.
- `504LM	_2` is 8 characters wide
- `# 504LM	_2` is 10 characters wide (consumes 2 characters of the next tab column)

> Line 76: Underscore used where there should be a negative

Typos present need to remain as they are in the scans - so here, we have to keep the underscore instead of a minus-dash.

> Line 115: Possible that '1ST' is typed 'IST', unsure

I've checked the scans and they did indeed type `IST` (capital `i`) instead of `1ST`

image

> Line 161: Unsure what character is used, left unchanged from '+'

This looks like an ampersand (&) with an older (but still used) letterform (🙲)

image

I've cross-referenced it with other ampersand occurrences in the transcriptions and the letterform matches the scans (and appear to be proper usage of an ampersand)

R0093 and R0095 https://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0003.jpg

image

**Comment by rcornbtree** [comment]
Thanks for the in-depth review and for making most of those changes ahead of me, I'll keep a better eye on leaving spacing + characters exactly as they are in future scans.

I've pushed another commit to fix the three comments that were out of scope from other changes, let me know if that's the right way to handle it -- open-source work on GitHub is a bit new to me still. Thanks!

File: Luminary099/GROUND_TRACKING_DETERMINATION_PROGRAM.agc [hunk 1]
```diff
@@ -116,8 +116,8 @@ P21PROG1	CAF	V6N34		# LOAD DESIRED TIME OF LAT-LONG.
 		SLOAD	SR1
 			OPTION2
 		BHIZ	SET
-			+2		# ZERO -- THIS VEHICLE (LM)
-			VINTFLAG	# ONE -- OTHER VEHICLE (CM)
+			+2		# ZERO--THIS VEHICLE (LM)
```
Comment (wopian) [inline Luminary099/GROUND_TRACKING_DETERMINATION_PROGRAM.agc:119]:
```suggestion
			+2		# ZERO--THIS VEHICLE(LM)
```

File: Luminary099/GROUND_TRACKING_DETERMINATION_PROGRAM.agc [hunk 2]
```diff
@@ -116,8 +116,8 @@ P21PROG1	CAF	V6N34		# LOAD DESIRED TIME OF LAT-LONG.
...
+			VINTFLAG	# ONE--OTHER VEHICLE (CM)
```
Comment (wopian) [inline Luminary099/GROUND_TRACKING_DETERMINATION_PROGRAM.agc:120]:
```suggestion
			VINTFLAG	# ONE--OTHER VEHICLE(CM)
```

**Review by wopian — APPROVED** [review]

**Comment by wopian** [comment]
> I've pushed another commit to fix the three comments that were out of scope from other changes, let me know if that's the right way to handle it -- open-source work on GitHub is a bit new to me still. Thanks!

That's the correct way to handle it 👍

**Pull request merged**